deployments/azure-kubernetes-service-helm.yml

```diff
@@ -0,0 +1,119 @@
+# This workflow will build and push an application to a Azure Kubernetes Service (AKS) cluster when you push your code
+#
+# This workflow assumes you have already created the target AKS cluster and have created an Azure Container Registry (ACR)
+# For instructions see:
+#   - https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal
+#   - https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal
+#   - https://github.com/Azure/aks-create-action
+#
+# To configure this workflow:
+#
+# 1. Set the following secrets in your repository (instructions for getting these 
+#    https://github.com/Azure/login#configure-a-service-principal-with-a-federated-credential-to-use-oidc-based-authentication):
+#    - AZURE_CLIENT_ID
+#    - AZURE_TENANT_ID
+#    - AZURE_SUBSCRIPTION_ID
+#
+# 2. Set the following environment variables (or replace the values below):
+#    - AZURE_CONTAINER_REGISTRY (name of your container registry / ACR)
+#    - RESOURCE_GROUP (where your cluster is deployed)
+#    - CLUSTER_NAME (name of your AKS cluster)
+#    - CONTAINER_NAME (name of the container image you would like to push up to your ACR)
+#    - SECRET_NAME (name of the secret associated with pulling your ACR image)
+#
+# 3. Choose the appropriate render engine for the bake step https://github.com/Azure/k8s-bake. The config below assumes Helm.
+#    Set your helmChart, overrideFiles, overrides, and helm-version to suit your configuration.
+#    - CHART_PATH (path to your helm chart)
+#    - CHART_OVERRIDE_PATH (path to your helm chart with override values)
+#
+# For more information on GitHub Actions for Azure, refer to https://github.com/Azure/Actions
+# For more samples to get started with GitHub Action workflows to deploy to Azure, refer to https://github.com/Azure/actions-workflow-samples
+# For more options with the actions used below please see the following:
+#   - https://github.com/Azure/login
+#   - https://github.com/Azure/aks-set-context
+#   - https://github.com/marketplace/actions/azure-cli-action
+#   - https://github.com/Azure/k8s-bake
+#   - https://github.com/Azure/k8s-deploy   
+
+name: Build and deploy an app to AKS with Helm
+
+on:
+  push:
+    branches:
+      - $default-branch
+  workflow_dispatch:
+
+env:
+  AZURE_CONTAINER_REGISTRY: "your-azure-container-registry"
+  CONTAINER_NAME: "your-container-name"
+  RESOURCE_GROUP: "your-resource-group"
+  CLUSTER_NAME: "your-cluster-name"
+  SECRET_NAME: "your-secret-name"
+  CHART_PATH: "your-chart-path"
+  CHART_OVERRIDE_PATH: "your-chart-override-path"
+
+permissions:
+  actions: write
```

Comment:
where is the actions: write permission being used?

Reply:
<img width="1768" alt="Screen Shot 2022-02-15 at 12 18 55 PM" src="https://user-images.githubusercontent.com/18180506/154114684-af820269-5437-49fb-86a7-133b2bbeccd9.png">

Here is an example.  The deployment in actuality does succeed, but on the GitHub side we receive an HttpError that would lead the user to believe something went wrong.

Reply:
@tbarnes94 have you tried with `actions:read` permission. I think that should work well.

Reply:
Just tried with `actions: read` and that worked.  Thanks.  Made the change.